app/assets/stylesheets/course/achievement.scss

```diff
@@ -6,8 +6,8 @@
   .achievement-list,
   .achievement-form {
     .image > img {
```

Comment:
Not sure yet, but I think I will have a better answer when I start working on #761 - maybe we can defer this till then?